src/Ssh/Ssh/Common/SshBaseCmdlet.cs

```diff
@@ -417,19 +485,23 @@ protected internal string GetCertificateExpirationTimes()
             return null;
         }
 
-        protected internal string GetSSHClientPath(string sshCommand)
+        protected internal string GetClientApplicationPath(string command)
         {
 
             if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
-                return sshCommand;
+                return command;
             }
 
-            sshCommand = $"{sshCommand}.exe";
-            var cmdInfo = InvokeCommand.GetCommand(sshCommand, CommandTypes.Application);
+            command = $"{command}.exe";
+            var cmdInfo = InvokeCommand.GetCommand(command, CommandTypes.Application);
```

Comment:
You are searching for a specific type of command (native application), so you should make sure that is what you get.
```suggestion
            ApplicationInfo appInfo = InvokeCommand.GetCommand(command, CommandTypes.Application) as ApplicationInfo;
```